src/main/java/com/mmm/his/cer/utility/farser/lexer/set/SetLogicTokenType.java

```diff
@@ -0,0 +1,55 @@
+package com.mmm.his.cer.utility.farser.lexer.set;
+
+import com.mmm.his.cer.utility.farser.CommonTokenFlag;
+import com.mmm.his.cer.utility.farser.ast.AstCommonTokenType;
+import com.mmm.his.cer.utility.farser.ast.AstTokenType;
+import com.mmm.his.cer.utility.farser.lexer.CommonTokenType;
+import com.mmm.his.cer.utility.farser.lexer.TokenType;
+import java.util.Optional;
+
+/**
+ * Token type for handling set logic.
+ *
+ * @author Mike Funaro
+ */
+public enum SetLogicTokenType implements TokenType<SetLogicTokenType>,
+    AstTokenType<SetLogicTokenType> {
+  ATOM(null, CommonTokenType.ATOM),
+  SPACE(" ", CommonTokenType.SPACE),
+  LPAREN("(", AstCommonTokenType.LPAREN),
+  RPAREN(")", AstCommonTokenType.RPAREN),
+  INTERSECTION("&", 2),
```

Comment:
The precedence _may_ also be the same for all. Unless it is needed or convenient to have different precedences

Reply:
Set logic does use this precedence order, I looked it up to be sure.